inc/deprecated.php

```diff
@@ -374,3 +374,53 @@ function get_rocket_ecommerce_exclude_pages() {
 		return array();
 	}
 endif;
+
+if ( ! function_exists( 'run_rocket_bot' ) ) :
```

Comment:
Why use `function_exists()` on your deprecated functions?

Reply:
I followed the pattern used for this file, if it's useless I can remove them

Reply:
I can't see in which cases it could be useful. But anyway, don't bother, it's not a problem, I was just curious.